frontend/components/common/InstantSearchProvider.tsx

```diff
@@ -315,3 +333,18 @@ function getBaseOrigin(location: Location): string {
    }
    return location.origin;
 }
+
+function getDevicePath(handle: string, routeState: RouteState): string {
+   const [device, variant] = handle.split(':');
```

Comment:
I just realized this should have been coded to handle a `:` in the variant text. Right now, it throws away the rest. We'll need to use `indexOf` or something cause `split(':', 2)` also throws away the rest of the string.

There are only 2 variants with colons in them, but I expect that could grow.

![image](https://github.com/iFixit/react-commerce/assets/26855/f92420db-acc5-4861-898a-fe3f82f0f083)